src/index.html

```diff
@@ -10,8 +11,176 @@
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="input-block">
+          <label for="surname" required>Surname:</label>
+          <input
+            type="text"
+            id="surname"
+            autocomplete="off"
+            name="surname"
+          >
+        </div>
+        <div class="input-block">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+          >
+        </div>
+        <div class="input-block">
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            name="age"
+            id="age"
+            value="12"
+            max="100"
+            min="9"
+          >
+        </div>
+        <div class="input-block">
+          <label for="birthday">Full date of birth:</label>
+          <input
+            type="date"
+            id="birthday"
+            name="birthday"
+            value=""
+            required
+          >
+        </div>
+        <div class="input-block">
+          <label for="agreement">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            id="agreement"
+            name="agree"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div class="input-block">
+          <label for="useremail">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="useremail"
+            placeholder="email@example.com"
+            autocomplete="off"
+          >
+        </div>
+        <div class="input-block">
+          <label for="userpassword">Password:</label>
+          <input
+            type="password"
+            id="userpassword"
+            name="password"
+            minlength="8"
+            maxlength="30"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="input-block">
+          Do you love cats?
+          <input
+            type="radio"
+            id="contactChoice1"
+            name="fact"
+            value="yes"
+          >
+          <label for="contactChoice1">Yes</label>
+
+          <input
+            type="radio"
+            id="contactChoice2"
+            name="fact"
+            value="no"
+          >
+          <label for="contactChoice2">No</label>
+        </div>
+
+        <div class="input-block">
+          <label for="color">What is your favorite color?</label>
+          <input
+            type="color"
+            id="color"
+          >
```

Comment:
If you have less than 3 attributes, you can fit them in one line
```suggestion
          <input type="color" id="color">
```